content/kb/tutorials/llm/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: LLM Tutorials
+slug: /knowledge-base/tutorials/llm-tutorials
+---
+
+# LLM Tutorial: OpenAI, LangChain, and Streamlit in 18 lines of code
+
+In this tutorial, you will build a Streamlit LLM app that can generate text from a user-provided prompt. This Python app will use the LangChain framework and Streamlit. Optionally, you can deploy your app to [Streamlit Community Cloud](https://streamlit.io/cloud) when you're done.
+
+*This tutorial is adapted from a [blog](https://blog.streamlit.io/langchain-tutorial-1-build-an-llm-powered-app-in-18-lines-of-code/) post by Chanin Nantesanamat.*
+
+<Cloud src="https://doc-tutorial-llm-18-lines-of-code.streamlit.app/?embed=true" height="600" />
+
+### Objectives
+
+1. Get an OpenAI key from the end user.
+2. Validate the user's OpenAI key.
+3. Get a text prompt from the user.
+4. Authenticate OpenAI with the user's key.
+5. Send the user's prompt to OpenAI's API.
+6. Get a response and display it.
+
+Bonus: Deploy the app on Streamlit Community Cloud!
+
+### Prerequisites
+
+- Python 3.7+
```

Comment:
bump to 3.8+ since we no longer support 3.7 as of last month

Reply:
Excellent note.